apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -35,7 +35,7 @@ export const PICKER_TYPE = makeEnum(
 
 /**
  * Dialog used for finding/selecting/uploading one or more assets to add to a
- * GameLab project.
+ * Game Lab/Sprite Lab project or curriculum levels.
```

Comment:
Your comment update is probably clearer anyway, but maybe this is supposed to be GameLab like the Rails model Gamelab and all of its children?